README.md

```diff
@@ -62,6 +71,45 @@ function configureBot(bot) {
 exports.configureBot = configureBot;
 ```
 
+```python
+import json
+import logging
+import threading
+import javascript
+
+from javascript import require, On
+
+mineflayer = require('mineflayer')
+mineflayer_pathfinder = require('mineflayer-pathfinder')
+rg_bot = require('rg-bot')
+rg_match_info = require('rg-match-info')
+Vec3 = require('vec3').Vec3
```

Comment:
we can move these into a different file and import them like regular Python imports, so that we don't need any of these `require` calls, and are therefore more Pythonic.

More explicitly, we create an `__init__.py` file inside of the module, and do
```python
from javascript import require, On

mineflayer = require('mineflayer')
mineflayer_pathfinder = require('mineflayer-pathfinder')
rg_bot = require('rg-bot')
rg_match_info = require('rg-match-info')
Vec3 = require('vec3').Vec3
```

And then in the user's file, all they need to do is
```python
from regression_games import mineflayer, mineflayer_pathfinder, rg_bot, rg_match_info, Vec3
```

Reply:
We can, but more advanced users are definitely going to need to use this require pattern to load in additional prismarine/mineflayer libraries at some point.

Reply:
I would err on putting just rg_bot and rg_match_info into this pattern since we own them

Reply:
also.. this isn't a python module, it is just a standalone launcher script, thus I think that in the user's bot code `from regression_games` wouldn't resolve to anything

Reply:
I see - let's leave this for now then, agree that importing other mineflayer libs will be something they may do often...